docs/reference/ImageDraw.rst

```diff
@@ -254,6 +254,22 @@ Methods
     :param outline: Color to use for the outline.
     :param fill: Color to use for the fill.
 
+
+.. py:method:: ImageDraw.regular_polygon(*, b_box, nb_sides, rotation=0, fill=None, outline=None)
```

Comment:
`thing(*, arg1, arg2)` means all the arguments after the asterisk are keyword-only ([PEP 3102](https://www.python.org/dev/peps/pep-3102/)), e.g. the user has to call `thing(arg1=x, arg2=y)`, they cannot call `thing(x, y)`.

We've not required that before, any particular reason to use it here?

Reply:
@hugovk , thanks for the review. keyword-only args are a personal preference but I've implemented your suggestion in order to maintain consistency with the rest of the codebase.